extensions/humaans/src/api/me.ts

```diff
@@ -0,0 +1,10 @@
+import { apiRequest } from "./api";
+
+export async function getMe(): Promise<{ id: string }> {
+  const {
+    data: { id },
+  } = await apiRequest({
+    endpoint: "me",
+  });
+  return { id };
+}
```

Comment:
This should seldom change. Thoughts on caching?


```suggestion

import { apiRequest } from "./api";
import { Cache } from "@raycast/api";

// Initialize the cache
const cache = new Cache();

export async function getMe(): Promise<{ id: string }> {
  // Check if the data is already in cache
  const cachedData = cache.get("me");
  
  if (cachedData) {
    // Parse the cached string back to an object
    return JSON.parse(cachedData);
  } else {
    const {
      data: { id },
    } = await apiRequest({
      endpoint: "me",
    });
    
    // Cache the new data using JSON.stringify to convert it into a string
    cache.set("me", JSON.stringify({ id }));
    
    return { id };
  }
}

```

Reply:
Yeh good idea. I've added the cache for this with the api key mixed in as the user is tied to that